.github/workflows/python.yml

```
@@ -46,6 +48,11 @@ jobs:
           curl -LOJs https://raw.githubusercontent.com/seqsense/ros_style/${{ inputs.style-branch }}/.flake8
           curl -LOJs https://raw.githubusercontent.com/seqsense/ros_style/${{ inputs.style-branch }}/pyproject.toml
 
+          extend_ignore="${{ inputs.flake8_extend_ignore }}"
+          if [ -z "${extend_ignore}" ]; then
```

```suggestion
          if [ -n "${extend_ignore}" ]; then
```
If I understand correctly, the `sed` line tries to add `${extend_ignore}` (`EXXX, EYYY, EZZZ...`) with a preceding comma at the end of the `ignore` line in `.flake8`. If correct, we want to execute the `sed`  if `${extend_ignore}` is not empty. What do you think?

---

Yes you're right, copy paste mistake :bow: